Assets/MixedRealityToolkit.Tests/EditModeTests/Core/MixedRealityToolkitFilesTests.cs

```diff
@@ -9,16 +9,18 @@
 using System;
 using System.IO;
 using System.Threading.Tasks;
+using System.Linq;
 
 namespace Microsoft.MixedReality.Toolkit.Tests.Core
 {
     // Tests for the MixedRealityToolkitFiles utility class
-    public class TestFixture_02_MixedRealityToolkitFilesTests
+    public class MixedRealityToolkitFilesTests
```

Comment:
Can you also rename the TestFixture_01_...?

Reply:
I don't understand why you would want to rename the test functions?  Keep to the standards used for all the other tests and revert these changes.  Only helper methods that are not tests should have such names.

Reply:
I can do the other rename separately - the existing naming of TestFixture_#_ adds no value and only clutters things up.

If the intent is specific order, that's a really bad intent - you should NEVER depend on your tests working on a specific order.

Reply:
@SimonDarksideJ I don't see that naming pattern being the standard though. It's only used by a handful of tests. The vast majority are more straightforward. "TestFixture" on the front seems redundant with adding "Tests" to the end of the name and the number doesn't seem to have any purpose or value. 

I agree with @wiwei and say we make the names simple like the other tests so it is more readable for others